samples/advanced-markers-accessibility/index.ts

```diff
@@ -54,11 +50,12 @@ async function initMap() {
         // [START maps_advanced_markers_accessibility_marker]
```

Comment:
at line 47, where we create the PinElement, let's not use the `glyph` property, let's use `glyphText`.